content/docs/card/case-study-card.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Case Study Card
+description: An interactive case study card that expands on hover, revealing a hidden tab with a clickable link to read comments.
+author: parankarj
+---
+
+<ComponentPreview name="card-case-study-card--docs" />
+
+## Installation
+
```

Comment:
<Steps> is missing here. Please check locally once!

Reply:
Im sorry, I was running only the story.